roofit/roofitcore/test/TestStatistics/testLikelihoodGradientJob.cpp

```diff
@@ -475,8 +483,10 @@ TEST_P(LikelihoodGradientJob, Gaussian1DAlsoWithLikelihoodJob)
 
    RooFit::MultiProcess::Config::setDefaultNWorkers(NWorkers);
    auto likelihood = std::make_shared<RooFit::TestStatistics::RooUnbinnedL>(pdf, data);
-   RooMinimizer m1(likelihood, RooFit::TestStatistics::LikelihoodMode::multiprocess,
-                   RooFit::TestStatistics::LikelihoodGradientMode::multiprocess);
+   RooMinimizer::Config cfg;
+   cfg.parallel_likelihood = true;
+   cfg.parallel_gradient = true;
+   RooMinimizer m1(likelihood, cfg);
```

Comment:
In this file, you are using all these functions that are now static, like set `setStrategy`, etc.

Can you change all these function calls here to actually make use of that staticness? Like `RooMinimizer::setStrategy(0)`.

Like this, it will also become apparent that many calls are redundant and you can delete quite a few lines :)